server-spi-private/src/main/java/org/keycloak/models/utils/ModelToRepresentation.java

```diff
@@ -788,6 +788,7 @@ public static ResourceServerRepresentation toRepresentation(ResourceServer model
         server.setName(client.getClientId());
```

Comment:
in the line :
server.setClientId(model.getId());
Shouldn't that be:
server.setClientId(client.getId());
?

Reply:
Both client and RS share the same ID.